### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by karanJ2212**
-Fetched data using API
-setup routers
-Stored the data in redux store
-displayed data from redux store
-styled the components
-deployed

### CODE REVIEW

File: src/Components/Weather/Weather.css [hunk 1]
```diff
@@ -91,7 +91,8 @@
 
 .loading {
   text-align: center;
-  font-size: 18px;
+  font-size: 3rem;
+  color: #000;
```
Comment (andersonlebon) [inline src/Components/Weather/Weather.css]: - [ ] Kindly make sure your design match with the template. kindly check the screencast below to check more about this

https://drive.google.com/file/d/1TrGdiVB9qDIBjOeE-VWC82ueqfu9BJPP/view?usp=sharing

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @karanJ2212 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

almost-there

## Suggested changes

- [ ] Kindly make sure The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

- [ ] kindly provide a video description(max 5minutes) that gives a short overview of  your App here is what should be included:

- a description of the project.
- a demo of the project features.
- you should also highlight some interesting piece of code or something you built that you are very proud of.

_Check the comments under the review._


You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @karanJ2212,

Great work on making the changes requested by the previous reviewer 👏

You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

# Required Changes ♻️

**As the previous reviewer suggested, Kindly do the following changes:**

- [ ] Kindly make sure The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

- [ ]  kindly provide a video description(max 5 minutes) that gives a short overview of your App here is what should be included:

    * a description of the project.

   * a demo of the project features.

   * you should also highlight some interesting piece of code or something you built that you are very proud of.

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "Fixed bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by TuyishimireEric** [comment]
# INVALID ❌ ❌ ❌
### Hi @karanJ2212   👋

While you made a great effort in this project, I can't go ahead and review your code.

## Invalid Code Review Request ❌
You have submitted a project that is not checking for Linter Errors 😨, Kindly configure Linters and submit again once you are done 🙏  

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Kindly create  a new PR that follows the Gitflow approach

Cheers, and Happy coding!👏👏👏

✅ Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

⚠️ Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

---
_Invalid Code Review Request does not count into the code reviews limit._

File: src/Components/Countrylist/Countrycard.css [hunk 2]
```diff
@@ -15,6 +13,22 @@
...
+
+.card:nth-child(4n+3) {
+  background: linear-gradient(rgba(209, 66, 121, 0.9), rgba(209, 66, 121, 0.9)), url('../../Images/image3.png') center/cover;
+}
+
+.card:nth-child(4n+4) {
+  background: linear-gradient(rgba(209, 66, 121, 0.9), rgba(209, 66, 121, 0.9)), url('../../Images/image4.png') center/cover;
+}
+
 .card-title {
```
Comment (Gambit142) [inline src/Components/Countrylist/Countrycard.css:32]: - [OPTIONAL] The first reviewer asked that you make your design identical to that of the design template provided for this project. You didn't implement it. I would advise you to click on the video link provided in this comment of the first reviewer https://github.com/karanJ2212/weather_report/pull/5#discussion_r1172773478 and implement the changes suggested. 👍 😎

**Review by Gambit142 — APPROVED** [review]
Hi @karanJ2212 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- Acknowledged the author of the original design ✔️ 
- README for the project is professional ✔️ 

### Optional Suggestion
- Kindly consider the comment tagged _**[OPTIONAL]**_ below the review summary 💻 🌵 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**